src/main/java/fi/hsl/transitdata/cancellation/Main.java

```diff
@@ -32,7 +32,7 @@ public static void main(String[] args) {
     }
     
     private static String getDigitransitDeveloperApiUri() throws Exception {
-        String digitransitDeveloperApiUri = "";
```

Comment:
NullPointerException will be thrown at row 50 should this be merged. I kinda like it though so fix row 50 instead to be `if digitransitDeveloperApiUri == null`